bin/ruby-build

```diff
@@ -1122,6 +1122,8 @@ normalize_semver() {
 # If a compatible Homebrew-installed OpenSSL version is found during
 # checking, Ruby will be linked to it and the check will return false.
 needs_openssl() {
+  [ "$RUBY_BUILD_BUILD_OPENSSL" = "true" ] && return 0
```

Comment:
1. Would a better name be "RUBY_BUILD_VENDOR_OPENSSL" so that it's clear that openssl will be vendored and to avoid saying "build" twice?
2. We generally allow any non-blank string to be a truthy value for an environment variable, so I would rewrite this check as:

```suggestion
  [ -z "$RUBY_BUILD_VENDOR_OPENSSL" ] || return 0
```

Reply:
Done, thanks